django_mock_queries/utils.py

```diff
@@ -1,6 +1,12 @@
 from datetime import datetime, date
 from django.core.exceptions import FieldError
-from unittest.mock import Mock
```

Comment:
The `from unittest.mock import Mock` line should stay as on the *master* branch.

Reply:
Done